fern/pages/api-definition/fern-definition/api-yml/environments.mdx

```diff
@@ -65,6 +65,17 @@ default-environment: Production
 ```
 </CodeBlock>
 
+<Note> By providing a default environment, the generated SDK will be setup to hit the default.</Note>
+
+## Base Path
+If you would like all of your endpoints to be prefixed with a path, use `base-path`. 
+
+In the example below, every endpoint is prefixed with a `/v1`: 
+```yaml api.yml
+name: api
+base-url: /v1
```

Comment:
nit: is this supposed to be `base-path`?

Reply:
thank you